packages/google-cloud-api-gateway/noxfile.py

```diff
@@ -405,24 +405,3 @@ def prerelease_deps(session):
     session.run("python", "-c", "import google.auth; print(google.auth.__version__)")
 
     session.run("py.test", "tests/unit")
-
-    system_test_path = os.path.join("tests", "system.py")
```

Comment:
@parthea  Are all these system tests supposed to be removed? I'm not finding a PR that intended to do this.